src/apply.c

```diff
@@ -709,7 +709,7 @@ use_leash(struct obj *obj)
                 spotmon ? y_monnam(mtmp) : l_monnam(mtmp));
             mtmp->mleashed = 1;
             obj->leashmon = (int) mtmp->m_id;
-            wakeup(mtmp, FALSE);
+            wakeup(mtmp, FALSE, FALSE);
```

Comment:
Why would a mimic remain cloaked after you slipping a leash around it? You know it's there.